planning_launch/launch/scenario_planning/lane_driving/motion_planning/motion_planning.launch.py

```diff
@@ -261,7 +261,7 @@ def generate_launch_description():
             ),
             DeclareLaunchArgument("use_surround_obstacle_check", default_value="true"),
             DeclareLaunchArgument(
-                "cruise_planner", default_value="obstacle_stop_planner"
+                "cruise_planner", default_value="obstacle_cruise_planner"
```

Comment:
back to the old one

Reply:
Fixed at [1ca3d93](https://github.com/tier4/autoware_launch/pull/418/commits/1ca3d93931e0750372370b45c5da20095edcefc3)